packages/react-native/ReactCommon/react/renderer/components/view/BaseViewProps.cpp

```diff
@@ -561,16 +561,10 @@ Transform BaseViewProps::resolveTransform(
     return transformMatrix;
   }
 
-  // transform is matrix
-  if (transform.operations.size() == 1 &&
-      transform.operations[0].type == TransformOperationType::Arbitrary) {
-    transformMatrix = transform;
-  } else {
-    for (const auto& operation : transform.operations) {
-      transformMatrix = transformMatrix *
-          Transform::FromTransformOperation(
-                            operation, layoutMetrics.frame.size);
-    }
+  for (const auto& operation : transform.operations) {
+    transformMatrix = transformMatrix *
+        Transform::FromTransformOperation(
+                          operation, layoutMetrics.frame.size, transform);
   }
```

Comment:
This now does an unnecessary multiplication for the 'transform' case, I'd prefer to leave this as is.